### PR TITLE
 #6889 lua_api.txt describes caching mechanism of minetest.place_schematic

### PR DESCRIPTION
**Pull request by Sokomine**

### CODE REVIEW

File: doc/lua_api.txt [hunk 1]
```diff
@@ -3229,6 +3229,10 @@ These functions return the leftover itemstack.
     * `force_placement` is a boolean indicating whether nodes other than `air` and
       `ignore` are replaced by the schematic
     * Returns nil if the schematic could not be loaded.
+  Warning: Once you have loaded a schematic from a file, it will be cached. Future calls
```
Comment (rubenwardy) [inline doc/lua_api.txt]: Please make this a bullet point and align with the above bullets

**Comment by rubenwardy** [comment]
This PR should be renamed to

> Lua API docs: Add warning that schematic placing is cached

Before merging

File: doc/lua_api.txt [hunk 2]
```diff
@@ -3229,6 +3229,10 @@ These functions return the leftover itemstack.
     * `force_placement` is a boolean indicating whether nodes other than `air` and
       `ignore` are replaced by the schematic
     * Returns nil if the schematic could not be loaded.
+    * **Warning**: Once you have loaded a schematic from a file, it will be cached. Future calls
+      will always use the cached version and the replacement list defined for it,
+      regardless of weather the file or the replacement list parameter have changed.
```
Comment (srifqi) [inline doc/lua_api.txt]: `regardless of whether the file or the replacement list parameter have changed.`

Comment (srifqi) [reply doc/lua_api.txt]: Seems like already fixed.

Comment (Sokomine) [reply doc/lua_api.txt]: Thanks! Rubenwardy saw it as well and told me directly.

**Pull request merged**